README.md

```diff
@@ -1,40 +1,39 @@
-# automation firebase project
+# bss-firebase
 
-`firebase emulators:start`
+## Introduction
 
-to deploy one set of functions:
-`firebase deploy --only functions:[namespace]`
+This repo name is a misnomer! What we actually have is a -- er -- _monorepo_ composed of two distinct deployments:
 
-to deploy just the website:
-`firebase deploy --only hosting`
-
-## config
-
-https://firebase.google.com/docs/functions/config-env
-
-config structure
-```json
-{
-  "airtable": {
-    "api_key": "",
-    "base_id": "",
-    "inbound_table": "",
-    "intake_table": "",
-    "reimbursements_table": "",
-    "volunteers_table": ""
-  },
-  "twilio": {
-    "auth_token": "",
-    "sid": ""
-  },
-  "slack": {
-    "token": "",
-    "channel_to_id": {}
-  }
-}
+- [www.bedstuystrong.com](https://bedstuystrong.com/) (`./public/*`)
+- *Cloud Functions for Slack* (`./functions/*`)
+
+## Prerequisites
+
+- airtable access
+- firebase access (prod and test)
+- github org membership
+- join #tickets (for test functions slack app)
+- join #automation (for... this)
```

Comment:
listed the things needed from an ops perspective. might make sense to excise but ymmv @cubeghost